apps/jobs/utils.py

```diff
@@ -0,0 +1,32 @@
+from rest_framework import status
+from rest_framework.response import Response
+from challenges.models import (
+    ChallengePhase,
+    Challenge,
+)
+from participants.models import (ParticipantTeam,)
+
+
+def get_challenge_object(challenge_pk):
+    try:
+        Challenge.objects.get(pk=challenge_pk)
+    except Challenge.DoesNotExist:
+        response_data = {'error': 'Challenge does not exist'}
+        return Response(response_data, status=status.HTTP_400_BAD_REQUEST)
+
+
+def get_challenge_phase_object(challenge_phase_pk):
```

Comment:
@TroJan Yeah sure. I think for this we can follow a convention `get_{app_name}_{model_name}_object`. What do you say ?